libs/ui/src/lib/notification/Toast/Toast.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+
+// eslint-disable-next-line import/no-cycle
+import { useToast } from './Toast.context';
+import { CloseIcon } from './icons';
+import Icon from './Toast.icon';
+
+import * as S from './Toast.style';
+
+import { ToastProps } from './Toast.props';
+
+const Toast = ({
+  message: { id, title, description, type, noIcon, autoClose, timeToClose },
+}: ToastProps) => {
```

Comment:
Recebendo de forma destruturada, vc não precisa receber o message, recebe somente as props:
```suggestion
const Toast = ({ id, title, description, type, noIcon, autoClose, timeToClose }: ToastProps) => {
```

Reply:
boa!